src/manifolds/ProbabilitySimplex.jl

```diff
@@ -1,5 +1,5 @@
 @doc raw"""
-    ProbabilitySimplex{n} <: AbstractDecoratorManifold{𝔽}
+    ProbabilitySimplex{n,closed} <: AbstractDecoratorManifold{𝔽}
```

Comment:
Maybe just philosophically (also for dispatch) we could also have
* `open` as true false, which does of course he same just on storing `!closed!`instead
* `boundary` which we could do with symbols `:Open` and `:Closed` where one of them ( for historic reasons probably `:Open`) is the default. This might make dispatching code on `ProbabilitySimplex{n,:Open}`more readable then in `ProbablilitySimplex{n, True}` remembering what the `true` stands for?

Reply:
Yes, I didn't think about readability. Using symbols is of course nicer. I will change that.

Reply:
I actually came up with the idea also only _while_ writing the `open=true|false` thing. I think for code-readability the symbols are a bit nicer, yes, though they allow for a bit more undefined cases.